app/controllers/orderSuccessCtrl.js

```diff
@@ -1,7 +1,14 @@
 "use strict";
 
-app.controller("orderSuccess", function($scope, $location){
-
-
+app.controller("orderSuccess", function($http, $location, RootFactory){
+
+	RootFactory.getApiRoot()
+	.then((root)=>{
+		$http.get(`${root.orders}`)
+		.then((obj)=>{
+			let count = obj.data[0].line_items.length;
```

Comment:
The way this is written, I think it's only showing the number of line items on the LAST order saved to the database, regardless of user. Which is fine if this is only supposed to be a partial fix, as the counting itself is working, but if it's supposed to be showing the **active** Customer's cart info then you'll have to adjust your request or filter it somehow.